selinux_policy_aws/README.md

```diff
@@ -1,69 +1,115 @@
 # SELinux Policy Workshop - AWS Version
 
-![ansible](img/Ansible-Tower-Logotype-Large-RGB-FullGrey-300x124.png)
+![selinux](img/200px-SELinux_logo.svg.png)
 
 `selinux_policy_aws` is a ansible playbook to provision RHEL nodes in AWS, for SELinux workshops. This playbook uses Ansible to wrap Terraform, for provisioning AWS infrastructure and nodes. To find more info about Terraform [check here](https://www.terraform.io/docs/providers/aws/index.html)
 
 These modules all require that you have AWS API keys available to use to provision AWS resources. You also need to have IAM permissions set to allow you to create resources within AWS. There are several methods for setting up you AWS environment on you local machine.
```

Comment:
These modules all require that you have AWS API keys to provision AWS resources.
Set IAM permissions to enable creation of resources within AWS.
Remove the comma after environment
the last "you" should be "your"